app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityPresenter.kt

```diff
@@ -0,0 +1,58 @@
+package org.oppia.android.app.devoptions
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.databinding.DataBindingUtil
+import org.oppia.android.R
+import org.oppia.android.app.activity.ActivityScope
+import org.oppia.android.app.drawer.NavigationDrawerFragment
+import org.oppia.android.databinding.DeveloperOptionsActivityBinding
+import javax.inject.Inject
+
+/** The presenter for [DeveloperOptionsActivity]. */
+@ActivityScope
+class DeveloperOptionsActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity
+) {
+  private lateinit var navigationDrawerFragment: NavigationDrawerFragment
+  private lateinit var binding: DeveloperOptionsActivityBinding
+
+  fun handleOnCreate() {
+    binding = DataBindingUtil.setContentView(
+      activity,
+      R.layout.developer_options_activity
+    )
+    setUpNavigationDrawer()
+    val previousFragment = getDeveloperOptionsFragment()
+    if (previousFragment != null) {
+      activity.supportFragmentManager.beginTransaction().remove(previousFragment).commitNow()
```

Comment:
Instead of removing this. If the condition is null than only `add` fragment.
WDYT?

Reply:
Yes, I think that is a better approach. I'll do it.

Reply:
Done